### PR TITLE
Add python 3.7 and 3.8 support

### PR DESCRIPTION
**Pull request by vernans**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,17 +3,18 @@ python:
...
+after_success:
+- coverage combine
+- coveralls
 cache:
   pip: true
-  directories:
-  - eggs/
 notifications:
   webhooks:
     secure: t+7Be/C8+jAZrZFYjcK6mHkYIrydKm/f8EZK4t2V+XA87ICTPl5UwSE7+7u8CaC5mLVrO02Y+I7+INIZ5XS3UMJrkHfPMhiwCi7EbW1Fk27DpNCloX9NyQoaKD29TcDxV6GTOOEJrLkB9aBJy6CRj2lQXkvlhAN+5YaoNiT5oJCXCotG7lpkrLwLOlcqXFlESuoMYJIy1u0KeztXlc7KFmKSxNa9FqGmuKZ8OWmQCeUpq66oNGoXbXgqJEtAFew7s/G8xcP6t2bOYf0vlHKtdAUd8J+6sMPiJAAwE3nlAwVDhn5hIvkr7RN3QDf9TlXpX4bfyIrGf/KnHRnmAAPLBrPMDVG5K/Dwwv+YssIKfX3iCKsnUSbgpMU3RKyPI9Tk0KhV/TqGjbD4Y8Qm4x3LAqkHxq55LcRUpFCeq6dwTY45XDxR9di1ZDHiPWb6gKrj69KYUciiCM9b/6+SdAax0VRDwa4Z+Q8FQBFk9Pwisx31TzZNqWKISrFSj1OlUDdhb/hAm1QR49Kprv5lhG2wM9lzRtdyP+xhr887YYDv+JtW8Ul/fxDGErcXbv+ip0gnVTkWT4ZuY8V67QDfhPNJkz8M3KWejDxoPE6E0+hKvCoFr8t41cYP3A+gQiAfup3UO22b1m3+NyOs6Dcoayh0ZLIqd/5LjVT16HHdUXZlDwQ=
```
Comment (sallner) [inline .travis.yml]: Here is no newline at EOF. marius was vaguely aware of some problems with that. Maybe that could be solved by the editor on save.

File: tox.ini [hunk 2]
```diff
@@ -13,7 +12,8 @@ passenv = MYSQL_*
 setenv =
   COVERAGE_FILE=.coverage.{envname}
 deps = mock
-       pytest
+       py27: pytest < 5.0
+       py{36,37}: pytest
```
Comment (sallner) [inline tox.ini]: How does this work with py38 ? Maybe just as a dependency to the other packages? To be explicit, I would suggest the following. 
```suggestion
       !py27: pytest
```

File: tox.ini [hunk 3]
```diff
@@ -25,17 +25,14 @@ deps = mock
        execnet > 1.4.0
 commands = py.test {envsitepackagesdir}/gocept/testdb --cov={envsitepackagesdir}/gocept/testdb --junitxml=junit-{envname}.xml {posargs}
 
-[testenv:coverage-clean]
-deps = coverage
-skip_install = true
-commands = coverage erase
-
 [testenv:coverage-report]
+basepython = python3.8
```
Comment (sallner) [inline tox.ini:29]: Future, yeah!

**Review by sallner — COMMENTED** [review]
Looks good, with minor comments.

**Review by icemac — APPROVED** [review]
LGTM.

**Pull request merged**